nimble/host/src/ble_hs_hci_evt.c

```diff
@@ -144,7 +144,7 @@ static const struct ble_hs_hci_evt_dispatch_entry *
 ble_hs_hci_evt_dispatch_find(uint8_t event_code)
 {
     const struct ble_hs_hci_evt_dispatch_entry *entry;
-    int i;
+    unsigned i;
```

Comment:
in general looks good but lets make those explicit "unsigned int"

Reply:
done and rebased